irispy/iris_tools.py

```diff
@@ -139,14 +143,18 @@ def get_iris_response(time_obs=None, pre_launch=False, response_file=None, respo
         response_version_set = True
     else:
         response_version_set = False
-        if response_file_set is False:
-            response_version = 4
-    if response_file_set+pre_launch+response_version_set > 1:
+    if pre_launch_set + response_file_set + response_version_set == 2 or \
+        pre_launch_set + response_file_set + response_version_set == 3:
```

Comment:
Why was this changed? It looks like it does the same job as the previous code except that it has to perform more checks...  Or was there some unexpected bug?

Reply:
Yeah, I understand. But simply using `pre_launch` would throw an error for some yet to be identified reason.

Reply:
I was wondering why too, but the new code seems to work just fine.

Reply:
How odd!  Is it just the use of ```pre_launch```?  For example does this work:
```python
if response_file_set + pre_launch_set + response_version_set > 1:
```
?

Reply:
Yeah, think it must have been the use of `pre_launch`, but I am not sure why it doesn't work the way it was intended to. It should work supposedly. But doesn't...